src/Merge/ExtraPackage.php

```diff
@@ -125,13 +127,20 @@ protected function loadPackage($json)
     /**
      * Merge this package into a RootPackage
      *
-     * @param RootPackage $root
+     * @param RootPackageInterface $top
      * @param PluginState $state
      */
-    public function mergeInto(RootPackage $root, PluginState $state)
+    public function mergeInto(RootPackageInterface $top, PluginState $state)
     {
-        $this->mergeRequires($root, $state);
-        $this->mergeDevRequires($root, $state);
+        if ($top instanceof RootAliasPackage)
```

Comment:
PluginState::getRootPackage() already does this unwrap. Did I mess something up in the refactoring patch that dropped usage of it in a key location?